packages/html/src/timedurationpicker/tests/timedurationpicker.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import { TimePicker } from '../../timepicker';
+
+const root = ReactDOM.createRoot(
+    document.getElementById('app') as HTMLElement
+);
+
+const styles = `
+    #test-area {
+        max-width: 660px;
+    }
+`;
+
+root.render(
+    <>
+        <style>{styles}</style>
+        <div id="test-area" className="k-d-grid k-grid-cols-2">
+
+            <span>TimePicker</span>
+            <span>TimePicker RTL</span>
+
+            <div>
+                <TimePicker placeholder="TimePicker..." />
```

Comment:
We should not use `<TimePicker />` here, we should use `<TimeDurationPicker />`

This is applicable for all test cases in this test file.